js/common/bootstrap/errors.js

```diff
@@ -286,6 +286,7 @@
     "ERROR_GRAPH_NO_INITIAL_COLLECTION" : { "code" : 1946, "message" : "no valid initial collection found" },
     "ERROR_GRAPH_REFERENCED_VERTEX_COLLECTION_NOT_USED" : { "code" : 1947, "message" : "referenced vertex collection is not part of the graph" },
     "ERROR_GRAPH_NEGATIVE_EDGE_WEIGHT" : { "code" : 1948, "message" : "negative edge weight found" },
+    "ERROR_GRAPH_VERTEX_COLLECTION_NOT_USED" : { "code" : 1949, "message" : "vertex collection not used in graph" },
```

Comment:
Is this really "not used in graph" or rather "not part of the graph"?
```suggestion
    "ERROR_GRAPH_VERTEX_COLLECTION_NOT_USED" : { "code" : 1949, "message" : "vertex collection not used in graph" },
```

Reply:
Thanks, will change that and make it more clear